earth_enterprise/src/server/geresetpgdb

```diff
@@ -138,39 +162,54 @@ elif [ $type == "soft" ]; then
       exit
       ;;
   esac
-elif [ $type == "restore" ]; then
-  if [ ! -n "$2" ]; then
+elif [ $type == "restore" ] || [ $type == "upgrade" ] || [ $type == "backup" ]; then
```

Comment:
Style: This looks like it wold work better as a `case "$type"` instead of an `if`.